Tools/i18n/pygettext.py

```diff
@@ -124,6 +124,15 @@
     --width=columns
         Set width of output to columns.
 
+    --omit-header
+        Don’t write header to file.
+
+        This is useful for testing purposes because it eliminates a source of
+        variance for generated .gmo files.
+
+        Note: Using this option will lead to an error if the resulting file
```

Comment:
I think this should be clarified. The extraction will work, it's the compilation or other manipulation that may not work